frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts

```diff
@@ -0,0 +1,21 @@
+import { Extension } from './extension';
+
+namespace ExtensionProperties {
+  export interface ClusterServiceVersionAction {
+    /** the kind this action is for */
+    kind: string; // should be K8sKind probably
```

Comment:
This at least needs to be a `K8sResourceKindReference`

Reply:
ack